healthtools/scrapers/base_scraper.py

```diff
@@ -225,43 +229,41 @@ def archive_data(self, payload):
         '''
         Upload scraped data to AWS S3
         '''
-        data_key = DATA_DIR + self.data_key
-        data_archive_key = DATA_DIR + self.data_archive_key
```

Comment:
@DavidLemayian that the key doesn't exsist 
![image](https://user-images.githubusercontent.com/18587173/29042341-36f95948-7bbf-11e7-843a-9046da43345b.png)